dowhy/gcm/feature.py

```diff
@@ -0,0 +1,251 @@
+""" This modules allows to estimate the feature relevance of direct parents. For this, Shapley values are utilized with
```

Comment:
I'm trying to understand why it is called feature relevance? Are you considering a trained model with features?

Or is it just the relevance of direct parents for a node? In that case, I may suggest a different name since `feature` is typically used for ML models. E.g., interventional_importance or some other name

Reply:
I see that the paper considered a ML model for explanation. Is it the case here too?

Reply:
The algorithm itself just expects a blackbox model to analyze, i.e. it can be used completely without causal context (e.g. simply a prediction model). However, if you use it in the causal context (e.g. plug in the causal mechanism that describes a node), then the analysis becomes causal and explains the parents. Therefore, we choose "feature_relevance" to be more agnostic of the input model.

Reply:
Thanks for the explanation. Yes, parents_relevance sounds good. And agree for the other methods which can be used later for ML models also.